<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab6Database</class>
 <widget class="QWidget" name="Tab6Database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,4,1,1">
     <item>
      <widget class="QDateTimeEdit" name="pDTEFrom">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2022</year>
         <month>6</month>
         <day>10</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="pDTETo">
       <property name="dateTime">
        <datetime>
         <hour>12</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2022</year>
         <month>6</month>
         <day>30</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBSearch">
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBDelete">
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,5">
     <item>
      <widget class="QTableWidget" name="pTWDatabase">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>날짜</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>조도</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>온도</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>습도</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="pHLChart"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
